Index: libkirk/kirk_engine.c
===================================================================
--- libkirk/kirk_engine.c	(revision 30)
+++ libkirk/kirk_engine.c	(working copy)
@@ -216,7 +216,7 @@
   
   //Set the key
   AES_set_key(&aesKey, key, 128);
-  AES_cbc_encrypt(&aesKey, inbuff+sizeof(KIRK_AES128CBC_HEADER), outbuff+sizeof(KIRK_AES128CBC_HEADER), size);
+  AES_cbc_encrypt(&aesKey, inbuff+sizeof(KIRK_AES128CBC_HEADER), outbuff/*+sizeof(KIRK_AES128CBC_HEADER)*/, size);
   
   return KIRK_OPERATION_SUCCESS;
 }
